provision.py

```diff
@@ -124,10 +130,20 @@ def main():
             "source %s\n" % (os.path.join(VENV_PATH, "bin", "activate"),),
         ])
 
-    # Switch current Python context to the virtualenv.
+    # Switch current Python context to the python3 virtualenv
+    activate_this = os.path.join(PY3_VENV_PATH, "bin", "activate_this.py")
+    execfile(activate_this, dict(__file__=activate_this))
+
+    run(["pip", "install", "--upgrade", "pip"])
```

Comment:
This line is needed because in older versions of pip, I get an error `Submodule 'typeshed' not updated` during `pip install` of mypy-lang.